README.md

```diff
@@ -28,6 +28,12 @@ To install with npm:
 npm install chosen-js
 ```
 
+To install with composer:
```

Comment:
Can you please capitalize `composer` and link it to https://getcomposer.org?

Reply:
done